agent/php_wrapper.h

```diff
@@ -154,11 +154,11 @@ extern nruserfn_t* nr_php_wrap_user_function(const char* name,
                                              size_t namelen,
                                              nrspecialfn_t callback TSRMLS_DC);
 
-extern nruserfn_t* nr_php_wrap_user_function_with_transience(
+extern nruserfn_t* nr_php_wrap_user_function_with_options(
     const char* name,
     size_t namelen,
     nrspecialfn_t callback,
-    nr_transience_t TSRMLS_DC);
+    nr_wrap_user_function_options_t TSRMLS_DC);
```

Comment:
Should `options` be passed by value or by reference? Also `nr_php_wrap_user_function_with_options` does not modify options. If that's the case shouldn't this be indicated with a const?

Reply:
fixed in [449afaa](https://github.com/newrelic/newrelic-php-agent/pull/730/commits/449afaacda9a9d734919944b3742d90d348eb3d9)